packages/iov-ledger-bns/src/ledgersimpleaddresswallet.ts

```diff
@@ -300,15 +307,26 @@ export class LedgerSimpleAddressWallet implements Wallet {
     return out;
   }
 
-  private buildLocalIdentity(bytes: PublicKeyBytes, label: string | undefined): LocalIdentity {
-    const pubkey: PublicKeyBundle = {
-      algo: Algorithm.Ed25519, // hardcoded until we support more curves in the ledger app
-      data: bytes,
+  private buildLocalIdentity(
```

Comment:
This all seems fine. I was going to think more about how we use chainId in ledger, bu not really worth it until we revive the project... code is good for now, we need to revisit this all anyway when we reboot the ledger project